Templates/colors.json

```diff
@@ -0,0 +1,4687 @@
+        { "name": "BoostOutOfControlInv", "type": "float32" },
+        { "name": "CameraID", "type": "object_reference" }
+      ]
+    },
+    "ObjGeneratorSpawner": {
+      "fields": [
+        { "name": "KillerSpeed", "type": "float32" },
+        { "name": "RechargeInterval", "type": "float32" },
+        { "name": "BurstDistance", "type": "float32" },
+        { "name": "LaunchesTimeBeforehand", "type": "float32" },
+        { "name": "KillerType", "type": "ObjGeneratorSpawner::KillerType" }
+      ]
+    },
+    "ObjBattleShipShutterSpawner": {
+      "fields": [
+        { "name": "Type", "type": "ObjBattleShipShutterSpawner::Type" }
+      ]
+    },
+    "ObjCatapultSpawner": {
+      "fields": [
+        { "name": "Height", "type": "float32" },
+        { "name": "UpTime", "type": "float32" },
+        { "name": "DownTime", "type": "float32" },
+        { "name": "ImpulsePower", "type": "float32" },
+        { "name": "TargetID", "type": "object_reference" }
+      ]
+    },
+    "ObjLightRoadSpawner": {
+      "fields": [
+        { "name": "PathType", "type": "ObjLightRoadSpawner::PathType" },
+        { "name": "InOutType", "type": "ObjLightRoadSpawner::InOutType" },
+        { "name": "LengthType", "type": "ObjLightRoadSpawner::LengthType" },
+        { "name": "UsageType", "type": "ObjLightRoadSpawner::UsageType" }
+      ]
+    },
+    "ObjMonolithSpawner": {
+      "fields": [
+        { "name": "Phase", "type": "float32" }
+      ]
+    },
+    "ObjPartitionSpawner": {
+      "fields": [
+        { "name": "Type", "type": "ObjPartitionSpawner::Type" },
+        { "name": "Length", "type": "float32" },
+        { "name": "DownMoveTime", "type": "float32" },
+        { "name": "DownMoveTimeUnderHead", "type": "float32" },
+        { "name": "DownTime", "type": "float32" },
+        { "name": "UpMoveTime", "type": "float32" },
+        { "name": "UpTime", "type": "float32" }
+      ]
+    },
+    "ObjShortWarpSpawner": {
+      "fields": [
+        { "name": "Type", "type": "ObjShortWarpSpawner::Type" },
+        { "name": "Scale", "type": "float32" }
+      ]
+    },
+    "ObjWarpHoleSpawner": {
+      "fields": [
+      ]
+    },
+    "ObjGlassFloorSpawner": {
+      "fields": [
+        { "name": "Type", "type": "ObjGlassFloorSpawner::Type" },
+        { "name": "Size", "type": "ObjGlassFloorSpawner::Size" }
+      ]
+    },
+    "ObjAquariumDoorSpawner": {
+      "fields": [
+        { "name": "UpSpeed", "type": "float32" }
+      ]
+    },
+    "ObjFishbushSpawner": {
+      "fields": [
+      ]
+    },
+    "ObjFountainSpawner": {
+      "fields": [
+        { "name": "FirstSpeed", "type": "float32" },
+        { "name": "OutOfControl", "type": "float32" }
+      ]
+    },
+    "ObjGlassBarrierSpawner": {
+      "fields": [
+        { "name": "ItemType", "type": "ObjGlassBarrierSpawner::ItemType" },
+        { "name": "NumItems", "type": "uint32" }
+      ]
+    },
```

Comment:
Parameters named based on in-game functionality and parameter names from other objects with similar functionality.